### PR TITLE
blueprint: Use new Quilt API

### PR DESCRIPTION
**Pull request by kklin**
The Quilt API now deploys containers directly.

### CODE REVIEW

File: redis.js [hunk 1]
```diff
@@ -1,36 +1,40 @@
...
+const { Container, allow } = require('@quilt/quilt');
 
 const port = 6379;
 const image = 'quilt/redis';
 
 /**
  * Creates a service with a master Redis instance.
  * @param {string} auth - The password for autheticating with Redis.
- * @return {Service} - The Redis master service.
+ * @return {Container} - The Redis master service.
```
Comment (kayousterhout) [inline redis.js]: service -> container?

Comment (kklin) [reply redis.js]: Thanks, I fixed a couple more renames

File: redis.js [hunk 2]
```diff
@@ -1,36 +1,40 @@
-const { Container, Service } = require('@quilt/quilt');
+const { Container, allow } = require('@quilt/quilt');
 
 const port = 6379;
 const image = 'quilt/redis';
 
 /**
- * Creates a service with a master Redis instance.
+ * Creates a a master Redis container.
```
Comment (kayousterhout) [inline redis.js]: extra "a"

Comment (kklin) [reply redis.js]: Sigh

**Pull request merged**